python/kwiver/vital/modules/module_helpers.h

```diff
@@ -7,9 +7,11 @@
 
 #include <string>
 
-bool is_python_library_loaded(const std::string& python_library_path);
+bool is_python_library_loaded( const std::string& python_library_path );
 bool load_python_library_from_env();
-bool load_python_library_from_interpretor(const std::string python_library_path);
+bool load_python_library_from_interpretor(
+  const std::string python_library_path );
```

Comment:
@mwoehlke-kitware It might be that I've spent too much time in Python land lately, but is this the correct formatting here? Too me the closing ) should be on the next line, but that's just me.

Reply:
Yes, it's correct. (Well, except it should be `std::string const`, but uncrustify doesn't currently know how to fix that.)

Splitting the closing brace/bracket/parenthesis when there is a break after the corresponding opening glyph *is* a code style, but not one we use in C++ code.

Reply:
Good to know, thanks!